pkg/plugins/autodiscovery/flux/utils_test.go

```diff
@@ -0,0 +1,35 @@
+package flux
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSearchFluxFiles(t *testing.T) {
+	f, err := New(
+		Spec{
+			RootDir: "testdata",
+		}, "", "")
+
+	require.NoError(t, err)
+
+	err = f.searchFluxFiles("testdata", defaultFluxFiles[:])
+	if err != nil {
+		t.Errorf("%s\n", err)
```

Comment:
You can use `t.Error()` here and it'll output a stringified version of the error with a newline.